test/.nycrc-mocha-webpack

```diff
@@ -0,0 +1,8 @@
+{
+  "all": true,
+  "include": [
+    "client/"
+  ],
+  "reporter": ["json"],
+  "clean": false
+}
```

Comment:
Because these 2 are very similar, I was hoping to have only one, then specify the `exclude`/`include` at the CLI level in `package.json`, but for some reason I have not figured out, I could not make these work... :(